src/topology/algebra/module/linear_pmap.lean

```diff
@@ -0,0 +1,161 @@
+/-
+Copyright (c) 2022 Moritz Doll. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Moritz Doll
+-/
+
+import linear_algebra.linear_pmap
+import topology.algebra.module.basic
+
+/-!
+# Linear operators over topological vector spaces
```

Comment:
In the theory of unbounded operators, one usually defines a linear operator exactly as the thing that is a `linear_pmap` in mathlib (see for instance Birman-Solomyak, Davies, and Weidmann). It might be different if one looks into books about spectral theory for bounded operators.

Reply:
Yes, in an unbounded operator context I agree that this is clear. But this doc is going to show up in a general context, so I think we should be more precise here.

Reply:
I've added 'partially defined' in the title and the main part of the docstring. Is it fine for you using "operator" as a short-hand for "partially defined linear operator" in the docstrings of the lemmas?

Reply:
"unbounded operator", maybe?